docs/source/internals/sandbox.rst

```diff
@@ -88,6 +88,15 @@ at: http://localhost:8000.  A sample superuser is installed with credentials::
     password: testing
 
 
+.. warning::
+
+    Sandbox site has Django Debug Toolbar integrated by default, which might
```

Comment:
Could you please adjust the grammar as follows? I also propose to disable by overriding `INTERNAL_IPS` which is simpler/quicker that removing it from middleware/installed apps:

> The sandbox has has Django Debug Toolbar enabled by default, which will affect its performance. You can disable it by setting \``INTERNAL_IPS\`` to an empty list in your local settings.

Reply:
Ok done.